### ISSUE TITLE
 fatal error: scene_rdl2/render/util/AtomicFloat.h: No such file or directory    11 | #include <scene_rdl2/render/util/AtomicFloat.h>       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

### ISSUE THREAD

**Issue by mrjod** [dreamworksanimation/openmoonray#144]
fatal error: scene_rdl2/render/util/AtomicFloat.h: No such file or directory
   11 | #include <scene_rdl2/render/util/AtomicFloat.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
![Uploading Screenshot 2024-03-23 202050.png…]()

**Comment by mday-dwa** [comment]
Hello @mrjod, please can you verify whether you included the --recurse-submodules flag in your git clone statement when you cloned the openmoonray top-level repository?
Also, is there any other pertinent information about your setup which might help us diagnose the issue?
Thanks!

**Comment by mrjod** [comment]
yes i did but i dont think the --recurse-submodules flag worked becuase i had to manually download the other submodules
